capstone/src/main/java/com/google/sps/servlets/StudentServlet.java

```diff
@@ -26,4 +30,19 @@ private static String convertToJsonUsingGson(Student student) {
     String json = gson.toJson(student);
     return json;
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // TODO: All commented steps below
+
+    // Get student object based on the logged in email
+
+    // Add club to logged in student's club list if requested
+
+    // Remove club from logged in student's club list if requested
+
+    // Update student information with edited content
```

Comment:
nit: remove the empty new lines between the comments. If you were trying to separate the steps out so it was clear I'd just number them instead.

Reply:
Done!